DESCRIPTION

```diff
@@ -46,6 +46,7 @@ Imports:
     tools,
     withr
 Suggests: 
+    base64enc,
```

Comment:
I have mixed feelings about this but since we are including `base64enc` in the suggests, I think `decode_base64()` should at the very list check if the packaged has been installed. (I have mixed feelings because it is a dependency for other packages, so I can't really think of a scenario where it wouldn't be available.)

Reply:
It is a function that will only run in production. It should be installed already since the package `bslib` depends on it. What other reservations do you have for adding it in suggests, other than checking in `decode_base64` a check if the package is installed?

I will add the check mentioned by you in the function.